package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Are `package.json` and `package-locak.json` necessary?

Reply:
no it's probably a mistake! will remove